container_pipeline/lib/settings.py

```diff
@@ -0,0 +1 @@
+from container_pipeline.lib.default_settings import *  # noqa
```

Comment:
To make linters ignore errors/warnings. In this case, we're import ``*``, which is not usually OK, but it works here.